src/pybind/mgr/nfs/export.py

```diff
@@ -451,20 +459,18 @@ def _get_export_dict(self, cluster_id: str, pseudo_path: str) -> Optional[Dict[s
         log.warning(f"No {pseudo_path} export to show for {cluster_id}")
         return None
 
-    @export_cluster_checker
     def get_export(
             self,
             cluster_id: str,
             pseudo_path: str,
-    ) -> Tuple[int, str, str]:
+    ) -> Dict[str, Any]:
+        self._validate_cluster_id(cluster_id)
         try:
             export_dict = self._get_export_dict(cluster_id, pseudo_path)
-            if export_dict:
-                return 0, json.dumps(export_dict, indent=2), ''
-            log.warning("No %s export to show for %s", pseudo_path, cluster_id)
```

Comment:
is this warning msg not necessary anymore?

Reply:
It seemed like a "false" warning to me.  If you'd like I can add back a log line here but I think it would make more sense as an info or debug. Sound OK?

Reply:
If it's a false warning then I think it would be OK converting it to an info i.e

Reply:
done. converted to info.